web-components/src/components/tabs/Tabs.ts

```diff
@@ -939,16 +957,6 @@ export namespace Tabs {
       this.compUniqueId && (this.tabsOrderPrefsArray = localStorage.getItem(this.compUniqueId)?.split(",") || []);
     }
 
-    private allElements: Array<String> = [];
-
-    private updateSelectedTabIndexOnClick(e: any) {
-      const index = this.allElements.indexOf(e.target.id);
-
-      if (index !== -1) {
-        this.updateSelectedTab(index);
-      }
-    }
-
```

Comment:
This code was causing couple of issues with Tabs component and isn't needed as```updateSelectedTab``` is already called by ```handleTabClick```